File: micro-service/content-service/domain/platform.go
package domain

import (
	"time"
)

type PlatformSiteType int

const (
	platformUnknown PlatformSiteType = iota
	PlatformTypeSite
	PlatformTypeCompany
	PlatformTypeSummary
)

type Platform struct {
	ID               string
	Name             string
	SiteURL          string
	PlatformSiteType PlatformSiteType
	IsEng            bool
	FaviconURL       string
	CreatedAt        time.Time
	UpdatedAt        time.Time
	DeletedAt        *time.Time
}

File: web/client-v2/src/shared/types/platform.ts
export type PlatformSiteType =
  | PlatformTypeUnknown
  | PlatformTypeSite
  | PlatformTypeCompany
  | PlatformTypeSummary;

type PlatformTypeUnknown = 0;
type PlatformTypeSite = 1;
type PlatformTypeCompany = 2;
type PlatformTypeSummary = 3;
